packages/canary/src/styles.css

```diff
@@ -74,6 +76,22 @@
     --error: var(--red);
     --warning: var(--orange);
     --ai: var(--purple);
+    --diff-add-content--: hsl(156, 28%, 10%);
+    --diff-del-content--: hsl(356, 28%, 11%);
+    --diff-add-lineNumber--: hsl(153, 40%, 14%);
+    --diff-del-lineNumber--: hsl(358, 39%, 17%);
+    --diff-plain-content--: var(--background);
+    --diff-expand-content--: var(--background);
+    --diff-plain-lineNumber--: var(--background);
+    --diff-plain-lineNumber-color--: hsl(0, 0%, 100%);
+    --diff-hunk-content--: hsl(218, 26%, 12%);
+    --diff-hunk-lineNumber--: hsl(218, 26%, 12%);
+    --diff-add-content-highlight--: hsl(152, 37%, 24%);
+    --diff-del-content-highlight--: hsl(0, 38%, 26%);
+    --diff-add-widget--: hsl(216, 80%, 65%);
+    --diff-add-widget-color--: hsl(0, 0%, 100%);
+    --diff-empty-content--: var(--background);
+    --diff-hunk-content-color--: hsl(0, 0%, 100%);
```

Comment:
Do we have any recommendations from Pixel point designs in terms of which color to use?

Reply:
these hsl colors are from pixel point